password_generator.py

```diff
@@ -0,0 +1,9 @@
+import random
+import string
+
+def generate_password(length=8):
```

Comment:
Let's also add arguments for the users to choose what type of characters they want on their password. For example, add options whether to include either numeric, alphabetic or with special characters; i.e. if the user chooses to include all 3 of them, then the password will include a mixture of all 3 types (this should be the default).